code/modules/surgery/organs/eyes.dm

```diff
@@ -112,14 +112,15 @@
 	desc = "An electronic device designed to mimic the functions of a pair of human eyes. It has no benefits over organic eyes, but is easy to produce."
 	origin_tech = "biotech=4"
 	status = ORGAN_ROBOT
+	var/EMP_flash_strength = 1
 
 /obj/item/organ/internal/eyes/cybernetic/emp_act(severity)
 	if(!owner || emp_proof)
 		return
 	if(prob(10 * severity))
 		return
 	to_chat(owner, "<span class='warning'>Static obfuscates your vision!</span>")
-	owner.flash_eyes(visual = TRUE)
+	owner.flash_eyes(intensity = EMP_flash_strength, visual = TRUE)
```

Comment:
One thing this will do is make normal robotic eyes take some more damage from lower strength EMPs - which is negligible because of the EMP anyways, but it might be good to reflect this in the changelog.

To clarify, the parent proc on internal organs already deals damage if the organ is robotic, so this will punish welding eyes a lot more on the lower strength EMP.